sim_3d_volumes.py

```diff
@@ -0,0 +1,103 @@
+""" Generate 3D map of molecules
+"""
+import os
+import numpy as np
+import torch
+import coords
+
+
+CUDA = torch.cuda.is_available()
+
+if CUDA:
+    Main_dir = "/scratch/ex-kdd-1/NicolasLegendre/Cryo/Data"
+else:
+    Main_dir = os.getcwd() + "\\Data\\"
```

Comment:
It sounds like we could make this more general by taking these lines out as I do not see anywhere else a `torch` or `CUDA` dependency. This type of check could be done in an external script.